interfaces/backends/backends.go

```diff
@@ -28,21 +31,41 @@ import (
 	"github.com/snapcore/snapd/interfaces/seccomp"
 	"github.com/snapcore/snapd/interfaces/systemd"
 	"github.com/snapcore/snapd/interfaces/udev"
-	"github.com/snapcore/snapd/release"
 )
 
-// append when a new security backend is added
-var All = []interfaces.SecurityBackend{
-	&systemd.Backend{},
-	&seccomp.Backend{},
-	&dbus.Backend{},
-	&udev.Backend{},
-	&mount.Backend{},
-	&kmod.Backend{},
-}
+var All []interfaces.SecurityBackend = backends()
+
+func backends() []interfaces.SecurityBackend {
+	all := []interfaces.SecurityBackend{
+		&systemd.Backend{},
+		&seccomp.Backend{},
+		&dbus.Backend{},
+		&udev.Backend{},
+		&mount.Backend{},
+		&kmod.Backend{},
+	}
+	featureLevel, summary := aa.ProbeKernel().Evaluate()
+
+	// This should be logger.Noticef but due to ordering of initialization
+	// calls, the logger is not ready at this point yet and the message goes
+	// nowehre. Per advice from other snapd developers, we just print it
```

Comment:
Typo 'nowehre'

Reply:
Thanks :-) I need to sync my vimrc and `:set spell` there :)